kzz7yh-a07504/cod-ve07v1_kzz7yh-a07504.xml

```diff
@@ -336,8 +340,8 @@
    </p>
                <p xml:id="kzz7yh-a07504-d1e641">
                   <lb ed="#V" n="7"/>Ad primum in oppositum cum dicis quod in deo est 
-      <lb ed="#V" n="8"/>per modum 
-      po<lb ed="#V" n="9" break="no"/>tentie etc. dico quod non opetet: quia cum productio persone 
+      <lb ed="#V" n="8"/>persona procedens per modum 
+      po<lb ed="#V" n="9" break="no"/>tentie etc. dico quod non opum: quia quia cum productio persone 
```

Comment:
The "opz" abbreviation would be better if the p had a line through the stem, but I think this should still be rendered as "oportet"

```suggestion
      po<lb ed="#V" n="9" break="no"/>tentie etc. dico quod non oportet: quia quia cum productio persone 
```